R/tt_pos_and_access.R

```diff
@@ -939,42 +1025,62 @@ setMethod("[", c("VTableTree", "numeric", "numeric"),
           function(x, i, j, ..., drop = FALSE) {
     ## have to do it this way because we can't add an argument since we don't
     ## own the generic declaration
-    keep_topleft <- list(...)[["keep_topleft"]] ## returns NULL if not presesnt
+    keep_topleft <- list(...)[["keep_topleft"]] ## returns NULL if not present
     keep_titles <- list(...)[["keep_titles"]] %||% FALSE
+    keep_footers <- list(...)[["keep_footers"]] %||% keep_titles
     reindex_refs <- list(...)[["reindex_refs"]] %||% TRUE
-    if(is.null(keep_topleft))
-        keep_topleft <- NA
-
+    
+    # browser()
     nr <- nrow(x)
     nc <- ncol(x)
     i <- .j_to_posj(i, nr)
     j <- .j_to_posj(j, nc)
 
     ##  if(!missing(i) && length(i) < nr) {
-    if(length(i) < nr) { ## already populated by .j_to_posj
-        keep_topleft <- isTRUE(keep_topleft)
+    if(length(i) < nr) {## already populated by .j_to_posj
+        if (is.null(keep_topleft)) keep_topleft <- FALSE
```

Comment:
why/how would `keep_topleft` ever be `NULL`? should that be `is.na`? If so that is equivalent to the isTRUE form above, which is what it was doing previously